cookbooks/cdo-nginx/recipes/default.rb

```diff
@@ -0,0 +1,61 @@
+apt_repository 'nginx' do
+  uri          'ppa:nginx/development'
+  distribution 'trusty'
+end
+
+apt_package 'nginx'
+
+run_unicorn = '/run/unicorn'
+directory run_unicorn do
+  user node[:current_user]
+  group node[:current_user]
+end
+
+%w(dashboard pegasus).each do |app|
+  socket_path = File.join run_unicorn, "#{app}.sock"
+  file socket_path do
+    action :delete
+    not_if { ::File.socket?(socket_path) }
+  end
+  node.override['cdo-secrets']["#{app}_sock"] = socket_path
+end
+
+# Get/create the SSL cert via the `ssl_certificate` cookbook resource
```

Comment:
Please add a comment that the SSL certificate is only set  if the appropriate Chef attributes are set and document what those are (ssl_key and ssl_cert)